_sass/custom/media-breakpoint.scss

```diff
@@ -30,7 +30,7 @@
     .page-header h1 {
         font-size: 3.5rem;
     }
-    .container h2 {
+    .content-constrained h2 {
```

Comment:
changed to this page class to not have it affect all components with 'container' class, such as CTAs